push-sdk-swift/AGDeviceRegistration.swift

```diff
@@ -71,12 +93,30 @@ public class AGDeviceRegistration: NSObject, NSURLSessionTaskDelegate {
             let clientInfoObject = AGClientDeviceInformationImpl()
         
             clientInfo(config: clientInfoObject)
-        
+            
+            // Check if config is available in plist file
+            if clientInfoObject.variantID == nil && NSBundle.mainBundle().objectForInfoDictionaryKey("variantID") != nil {
+                clientInfoObject.variantID = NSBundle.mainBundle().objectForInfoDictionaryKey("variantID") as? String
+            }
+            
+            if clientInfoObject.variantSecret == nil && NSBundle.mainBundle().objectForInfoDictionaryKey("variantSecret") != nil {
+                clientInfoObject.variantSecret = NSBundle.mainBundle().objectForInfoDictionaryKey("variantSecret") as? String
+            }
+            
+            if self.serverURL?.absoluteString == nil && NSBundle.mainBundle().objectForInfoDictionaryKey("serverURL") != nil {
+                if let url = NSURL(string: (NSBundle.mainBundle().objectForInfoDictionaryKey("serverURL") as! String)) {
+                    self.serverURL = url
+                } else {
+                    assert(self.serverURL.absoluteString != nil, "'serverURL' should be set")
+                }
```

Comment:
these three are the static one for the AGClientDeviceInformation configuration, so it's fine to ignore alias and others here.

However, with lacking others (e.g. alias/categories) here, this metadata can be still updated (and sent to the server) in an app that uses this library, right ?